### PR TITLE
[JENKINS-69461] Setup wizard URL and password field use old form design

### PR DESCRIPTION
**Pull request by Langer0416**
See [JENKINS-69461](https://issues.jenkins.io/browse/JENKINS-69461).

### Before

Before

URL Before

### After

After

URL after

### Proposed changelog entries

Use jenkins-input replace old form design in Setup wizard page

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  - Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content-Security-Policy directives (see [documentation on jenkins.io](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7054"></a>

### CODE REVIEW

**Review by timja — APPROVED** [review]
non blocking issues:

----

for consistency may as well update the instance configuration page as well?

image

----

there's now a scroll bar (on 13" macbook at least) due to the increased size required, could reduce the heading spacing as it's a bit excessive currently

image

With validation errors it gets pushed off the page:

image

File: war/src/main/less/pluginSetupWizard.less
Comment (timja) [inline war/src/main/less/pluginSetupWizard.less:107]: what's this for?

Comment (timja) [inline war/src/main/less/pluginSetupWizard.less:107]: I see it does this:

image

seems unrelated to the PR?

Comment (Langer0416) [reply war/src/main/less/pluginSetupWizard.less:107]: ### Before
Error Before

### After
Error After

I attached the before and after screenshot, I found that it's use red border in the `Signin Page` when you enter wrong username or password (you can see it below), I'm not sure it's neccessary here or not, if it's unnecessary, i'll remove it later.



Signin png

**Comment by Langer0416** [comment]
> could reduce the heading spacing as it's a bit excessive currently

I have reduced some space here, please have a look at it, thanks.

### Before
Space before

### After
Space After

**Comment by daniel-beck** [comment]
The new top margin makes the "Create Admin User" and "Instance Configuration" views inconsistent.

In general this looks great 👍

**Comment by Langer0416** [comment]
> When I last reviewed it yes it looked like it should, but that should be applied across all setup wizard views for consistency

I have made all setup wizard page's heading spacing consistency, please have a look at it :eyes: , thanks.

File: core/src/main/resources/jenkins/install/SetupWizard/setupWizardConfigureInstance.jelly
Comment (timja) [inline core/src/main/resources/jenkins/install/SetupWizard/setupWizardConfigureInstance.jelly:37]: could this be made consistent with the other pages as well please? (so that the label is on top of the form inputs)

image

the table layout would need replacing with divs most likely

**Review by timja — APPROVED** [review]
no blockers would be great to make the jenkins url page layout consistent though

**Review by NotMyFault — APPROVED** [review]
Looks fine so far.

**Comment by NotMyFault** [comment]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
